### PR TITLE
Prepare for the official VM

### PR DESCRIPTION
**Pull request by jospoortvliet**
Feedback please @enoch85 @jnweiger @Kawohl ;-)

It looks like this now:
digits1
We will need documentation, it now links to https://doc.owncloud.org/server/8.1/admin_manual/installation/appliance_installation.html  which is rather empty. One of you editing https://github.com/owncloud/documentation/blob/master/admin_manual/installation/appliance_installation.rst  would be helpful ;-)

### CODE REVIEW

**Comment by jospoortvliet** [comment]
@karlitschek fyi

**Comment by enoch85** [comment]
Looks OK to me :+1: 

@jospoortvliet Could you please add `_ownCloud VM on Ubuntu 14.04 with MySQL and Apache: https://www.en0ch.se/pre-configured-owncloud-installaton/` to the doc.owncloud.org as well?

File: install-instructions.php [hunk 1]
```diff
@@ -87,37 +87,39 @@
...
 			    <div class="row">
-                              <div class="col-md-3">
+                              <div class="col-md-3 vmlist">
 			        <a target="_blank" style="margin: 0 14px 0 7px;" href="https://www.en0ch.se/owncloud_/pre-configured-owncloud-installaton/"><img width=113 src="<?php echo get_template_directory_uri(); ?>/assets/img/clouds/en0ch.png"></img></a>
 			      </div>
-			      <div class="col-md-9">
-			        This pre-configured VM runs on Ubuntu 14.04 with Apache and MySQL, and is <a href="https://www.en0ch.se/machine-setup/" target="_blank">fully configured from start</a>. Updates are managed through a built in script, and could be setup as a cron job for automatic updates. 
+			      <div class="col-md-9 vmlist">
+			        <a href="https://www.en0ch.se/machine-setup/" target="_blank">Pre-configured Ubuntu 14.04 Vmwith Apache and MySQL</a>. Managers updates with script, can be automated.
 			      </div>
```
Comment (enoch85) [inline install-instructions.php]: What happened here? :)

**Comment by karlitschek** [comment]
:+1:

File: install-instructions.php (hunk 1 above)
Comment (jospoortvliet) [inline install-instructions.php]: rewrote some stuff and I guess the diff tool just puked all over it ;-)

But it looks fine, so... ship it :package:

**Comment by jospoortvliet** [comment]
@enoch85 the documentation is in a different repo and dude, you are SO welcome to just go and edit the doc... You don't even have to check it out, you can do it right on github. ;-)

File: install-instructions.php (hunk 1 above)
Comment (enoch85) [inline install-instructions.php]: Vmwith
Managers

;)

**Comment by enoch85** [comment]
@jospoortvliet Yeah, I would if I could find the specific page. :)

**Comment by jnweiger** [comment]
@enoch  I guess the link is in the first description. go for it!

**Comment by enoch85** [comment]
@jnweiger You mean this: https://doc.owncloud.org/server/8.1/admin_manual/installation/appliance_installation.html ? I mean the GitHub file so that I can edit.

**Comment by jospoortvliet** [comment]
@enoch85 I put that in the text below the image in the very first post on this page: https://github.com/owncloud/owncloud.org/pull/676#issue-105845137

it is https://github.com/owncloud/documentation/blob/master/admin_manual/installation/appliance_installation.rst -> direct editing is cool.
Proof:
updating owncloud documentation optimized

**Comment by enoch85** [comment]
@jospoortvliet Yeah, I know how to edit. But the text is already there, why is it not shown in the https://doc.owncloud.org/server/8.1/admin_manual/installation/appliance_installation.html?

image

**Comment by jospoortvliet** [comment]
@enoch85 that takes some deployment. Make the changes first as a PR, then we merge, then I backport, then we deploy ;-)

Once it is all ready to go...

**Comment by jospoortvliet** [comment]
@enoch85 and with 'changes' I mean - what users have to do when they download the official VM, which should have its own section on top. You can do it, I can do it - all good with me.

**Comment by jospoortvliet** [comment]
To change on the download page: OVA as default for virtualbox

**Comment by jospoortvliet** [comment]
Ok, this is good to go now. Will merge before we put 8.1.3 live, then we can update the URL's in strings.php, then - it can go live. Yay ;-)

**Pull request merged**